src/views/html/about.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Media Catalogue - About</title>
+        <link rel="stylesheet" href="../../../src/views/css/about_styles.css">
+        <link rel="stylesheet" href="../../../src/views/css/blue_theme_styles.css" id="blueTheme">
+        <link rel="stylesheet" href="../../../src/views/css/red_theme_styles.css" id="redTheme">
+        <link rel="stylesheet" href="../../../src/views/css/green_theme_styles.css" id="greenTheme">
+        <script src="../../../src/views/js/utilities.js"></script>
+    </head>
+    <body onload="readSettings()">
+        <header>
+            <a id="logo" href="home.html"><img id="logoImage"src="../../../images/MediaCatalogueHeader[150].png" alt="Media Catalogue logo"></a>
+            <a class="tabs" href="home.html">Home</a>
+            <a id="activeTab" class="tabs" href="about.html">About</a>
+            <a class="tabs" href="catalogue.html">Catalogue</a>
+            <a class="tabs" href="web_settings.html">Web-Settings</a>
+        </header>
+        <div align="center">
+            <img id="aboutImage" src="../../../images/MediaCatalogueHeader[150].png" alt="Media Catalogue logo">
+            <p>This is the first paragraph about the Media Catalogue.</p>
+            <p>This is the 2nd paragraph for describing the Media Catalogue.</p>
+            <p>This is the THIRD paragraph that describes the media catalogue but I don't know what else it needs to contain.</p>
+        </div>
+        <footer>
+            <h6 id="footerLabel">&copy 2017 Media Catalogue</h6>
```

Comment:
It is good practice to have all HTML Entities in the following syntax "&entityName;". It will work without the semi-colon but rather put it in.